cirq/circuits/circuit_test.py

```diff
@@ -297,6 +298,48 @@ def test_empty_moments():
         use_unicode_characters=False,
         transpose=True)
 
+def test_symbol_addition_in_gate_exponent():
+    # 1-qubit test
+    qubit = cirq.NamedQubit('a')
+    circuit = cirq.Circuit.from_ops(
+        cirq.X(qubit)**0.5,
+        cirq.YPowGate(
+            exponent=sympy.Symbol('a') + sympy.Symbol('b')).on(qubit)
+    )
+    cirq.testing.assert_has_diagram(circuit,
+                                    "a: ───X^0.5───Y^Symbol(\"a + b\")───",
```

Comment:
Use an r-string to avoid the backslashes. Or use single-quotes.

Reply:
Done.